src/api/include/migraphx/migraphx.h

```diff
@@ -197,6 +203,43 @@ migraphx_status migraphx_parse_onnx_buffer(migraphx_program_t* out,
                                            size_t size,
                                            migraphx_onnx_options_t options);
 
+migraphx_status migraphx_op_names_destroy(migraphx_op_names_t op_names);
+
+migraphx_status migraphx_op_names_create(migraphx_op_names_t* op_names);
+
+migraphx_status migraphx_op_names_add(migraphx_op_names_t op_names, const char* name);
+
+migraphx_status migraphx_op_names_size(size_t* out, migraphx_op_names_t op_names);
+
+migraphx_status migraphx_op_names_get(const char** out, migraphx_op_names_t op_names, size_t idx);
+
+migraphx_status migraphx_quantize_fp16(migraphx_program_t prog, migraphx_op_names_t name);
+
+migraphx_status migraphx_quantize_fp16_default(migraphx_program_t prog);
+
+migraphx_status migraphx_calibration_data_destroy(migraphx_calibration_data_t calibration_data);
+
+migraphx_status migraphx_calibration_data_create(migraphx_calibration_data_t* calibration_data);
+
+migraphx_status migraphx_calibration_data_size(size_t* out,
+                                               migraphx_calibration_data_t calibration_data);
+
+migraphx_status migraphx_calibration_data_get(migraphx_program_parameters_t* out,
+                                              migraphx_calibration_data_t calibration_data,
+                                              size_t idx);
+
+migraphx_status migraphx_calibration_data_add_element(migraphx_calibration_data_t calibration_data,
+                                                      migraphx_program_parameters_t elem);
+
+migraphx_status migraphx_quantize_int8(migraphx_program_t prog,
+                                       migraphx_target_t target,
+                                       migraphx_calibration_data_t data,
+                                       migraphx_op_names_t name);
+
+migraphx_status migraphx_quantize_int8_default(migraphx_program_t prog,
```

Comment:
Maybe we should add a struct to store `migraphx_calibration_data_t` and `migraphx_op_names_t`. This way when we add more options we dont need to add new functions.

Reply:
added an structure quantize_options to include op_names and calibration_data.